packages/docs/pages/connections/flickr.md

```diff
@@ -0,0 +1,21 @@
+# Flickr
+
+:::info This page explains the steps you need to follow to set up the Flickr
+connection in Automatisch. If any of the steps are outdated, please let us know!
+:::
+
+1. Go to the [link](https://www.flickr.com/services/apps/create/) to **create an
+   app** on Flickr API.
+2. Click **Request an API key**.
+3. Apply for a non-commercial key.
+4. Fill the field of **What is the name of your app?**.
+5. Fill the field of **What are you building?**.
+6. Check the checkboxes.
+7. Click on **Submit** button.
+8. Copy **Key** and **Key Secret** values and save them to use later.
+9. Click **Edit auth flow for this app** to configure "Callback URL".
+10. Copy **OAuth Redirect URL** from Automatisch and paste it to the **Callback URL** field.
+11. Click **Save changes**.
+12. Paste **Key** and **Secret** values you have saved from the 8th step and paste them into Automatisch as **Consumer Key** and **Consumer Secret**, respectively.
+13. Click **Submit** button on Automatisch.
+14. Now, you can start using the Flickr connection with Automatisch. **Consumer Key** and **Consumer Secret**, respectively.
```

Comment:
We need to add this page to the vitepress config file. It's going to be manual until Typeform is also added. If then, we can manually include all the files inside of the connections folder.